rails.md

```diff
@@ -38,66 +44,76 @@ Answer the following questions. First, without external resources. Challenge you
 
 
 /users/       method="GET"     # :controller => 'users', :action => 'index'
+Will show you all users in the index through the method get
 
 /users/1      method="GET"     # :controller => 'users', :action => 'show'
+Will get you the user with id of 1
 
 /users/new    method="GET"     # :controller => 'users', :action => 'new'
+Will get you the user with the newest id
 
 /users/       method="POST"    # :controller => 'users', :action => 'create'
+Will create a whole new user
 
 /users/1/edit method="GET"     # :controller => 'users', :action => 'edit'
+Will bring up an editable information for user 1
 
 /users/1      method="PUT"     # :controller => 'users', :action => 'update'
+Will update the user with the id of 1
 
 /users/1      method="DELETE"  # :controller => 'users', :action => 'destroy'
-
+Will delete the user with the id of 1
 
 
 3b. Which of the above routes must always be passed params and why?
 
+Show, edit, update, and destroy. Because they all need a specific user id (params[:id]) so that they can manipulate.
 
 
 4. What is the public folder used for in Rails?
 
-  Your answer:
+  Your answer: Mostly errors, and boiler plates.
 
-  Researched answer:
+  Researched answer: In many MVC frameworks (e.g. Rails), the public folder is used to serve static files. So for example, you often see 404
 
 
 
 5. Write a rails route for a controller called "main" and a page called "game" that takes in a parameter called "guess"
 
+  get '/main' => '/game#guest'
 
```

Comment:
Close. It is asking for a page called game and parameter called guess so the route would be '/game/:guess' => '/main#game'